packages/core/src/controllers/verify.ts

```diff
@@ -62,40 +63,29 @@ export class Verify extends IVerify {
 
   public register: IVerify["register"] = async (params) => {
     if (!isBrowser()) return;
+    const origin = window.location.origin;
     const { id, decryptedId } = params;
-    const url = `${this.verifyUrlV2}/attestation?projectId=${this.core.projectId}`;
-    let src = "";
-    try {
-      const response = await fetch(url, {
-        method: "POST",
-        body: JSON.stringify({ id, decryptedId }),
-      });
-      const { srcdoc } = await response.json();
-      src = srcdoc;
-      this.logger.debug("srcdoc fetched", src);
-    } catch (e) {
-      this.logger.warn(e);
-      return;
-    }
+    const src = `${this.verifyUrlV3}/attestation?projectId=${this.core.projectId}&origin=${origin}&id=${id}&decryptedId=${decryptedId}`;
     try {
       const document = getDocument() as Document;
-      const abortTimeout = this.startAbortTimer(ONE_SECOND * 3);
+      const abortTimeout = this.startAbortTimer(ONE_SECOND * 5);
       const attestationJwt = await new Promise((resolve) => {
         const abortListener = () => {
           window.removeEventListener("message", listener);
           document.body.removeChild(iframe);
           throw new Error("attestation aborted");
```

Comment:
@ganchoradkov don't we need to `reject()` the promise here not just throw?

Reply:
The behaviour is the same whether is there is an error or the promise is rejected, just semantics